### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by IhorPokr**
- [DEMO LINK](https://IhorPokr.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://IhorPokr.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Comment by IhorPokr** [comment]
I don't understand why mine - [DEMO LINK](https://IhorPokr.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://IhorPokr.github.io/layout_moyo-header/report/html_report/) do not work

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
Your demo doesn't work as well, run `npm test` and `npm run deploy`

**Comment by IhorPokr** [comment]
> Your demo doesn't work as well, run `npm test` and `npm run deploy`

Do I need to make a new pool request?

File: src/style.css [hunk 1]
```diff
@@ -1 +1,67 @@
...
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: block;
+  line-height: 60px;
+  text-decoration: none;
+  font-style: normal;
```
Comment (alexandra-protyanova) [inline src/style.css:42]: it is default value u can delete it

**Review by alexandra-protyanova — APPROVED** [review]
GJ!

**Pull request closed without merging**